packages/frontend/components/student-journey/step-two.js

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback } from 'react';
+import { useDropzone } from 'react-dropzone';
+import StepTitle from './step-title';
+import Paragraph from '../ui/paragraph';
+import Card from '../ui/card';
+import DropBox from '../ui/drop-box';
+import IdCardIcon from '../../icons/id-card';
+
+function StepTwo() {
+  const onDrop = useCallback((acceptedFile, type) => {
+    // todo: change console.log after backend is ready.
+    // eslint-disable-next-line no-console
+    console.log(acceptedFile, type);
```

Comment:
same?

Reply:
same answer without formik. there is no way to check right file is selected or not because i can't upload it yet. i will delete it after back end is ready.

Reply:
why not use https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en ?